URLMock/Categories/NSURL+UMKQueryParameters.h

```diff
@@ -83,6 +89,10 @@
  @param baseURL The base URL for the NSURL object.
  @result An NSURL object
  */
-+ (instancetype)umk_URLWithString:(NSString *)URLString parameters:(NSDictionary *)parameters relativeToURL:(NSURL *)baseURL;
++ (_Nullable instancetype)umk_URLWithString:(NSString *)URLString
+                                 parameters:(NSDictionary * _Nullable)parameters
+                              relativeToURL:(NSURL * _Nullable)baseURL NS_SWIFT_NAME(init(string:parameters:relativeToURL:));
```

Comment:
Does this work correctly with ARC? The ObjC variant would return an autoreleased instance, but shouldn't the swift init result in ownership being transferred to the caller? I may be overthinking this…